Render/Render.as

```diff
@@ -70,18 +70,13 @@ void RenderWindows(){
     bool showRefreshButton = false;
 
     if (!UI::IsOverlayShown()) {
-        windowFlags |= UI::WindowFlags::NoInputs;
+        windowFlags |= UI::WindowFlags::NoMove;
```

Comment:
Just thought of this, but with this flag can you still move the window around when the OP menu is open (f3 to have the op banner etc)

Reply:
I just checked to make sure, it's movable only when the F3 menu is open, and not movable when the menu is closed.

Reply:
Perfect! Thanks for checking!